### PR TITLE
BaSP-TG-105: Time-Sheet form updated

### PR DESCRIPTION
**Pull request by martinlotufo93**

### CODE REVIEW

File: src/components/TimeSheet/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (GuidoCerioni) [inline src/components/TimeSheet/index.js:1]: you shouldn't desactivate rules, try to correct your eslint errors

File: src/components/TimeSheet/index.js [hunk 2]
```diff
@@ -136,98 +155,151 @@ const TimeSheetForm = () => {
...
+  if (loading) {
+    return (
+      <div className={styles.loading}>
+        <LoadingScreen />
+      </div>
+    );
+  }
   return (
     <div className={styles.container}>
       <h3>{requestType === 'PUT' ? 'Update Time Sheet' : 'Add Time Sheet'}</h3>
```
Comment (GuidoCerioni) [inline src/components/TimeSheet/index.js:167]: i think this h3 should have  color: #373867

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Nice job Martin!! Please check this two little things

**Review by AlejandroSantini — DISMISSED** [review]
Good job 👌

File: src/components/TimeSheet/form.module.css [hunk 3]
```diff
@@ -33,3 +33,16 @@
...
+  padding: 0;
+  list-style: none;
+}
+.li {
+  display: flex;
+  flex-direction: column;
+  margin: 0 0 20px 0;
+  align-items: center;
+}
+h3 {
```
Comment (Gearyandres94) [inline src/components/TimeSheet/form.module.css:46]: This should be a class something like .h3tittle or something like that.

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
Good job! A little thing to correct.

**Review by GuidoCerioni — APPROVED** [review]

**Review by tobiaspujol — APPROVED** [review]
changes suggested are corrected properly!

**Pull request merged**